issue-template.md

```diff
@@ -0,0 +1,14 @@
+Your Name: [Your name / twitter handle]
```

Comment:
No PR required, methinks. Just an open issue.

On Wed, May 25, 2016 at 11:09 AM Jordan Kasper <notifications@github.com>
wrote:

> In issue-template.md
> <https://github.com/dcjs/talks/pull/1#discussion_r64590504>:
>
> > @@ -0,0 +1,14 @@
> > +Your Name: [Your name / twitter handle]
>
> "&" versus "/" ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dcjs/talks/pull/1/files/5b0e767e0a6f4da691ba521a9ce3b69bdde3cae3#r64590504>
>